pkg/upgrade/config.go

```diff
@@ -14,6 +14,7 @@ type Config struct {
 	TargetCluster     TargetClusterConfig     `json:"targetCluster"`
 	KubernetesVersion string                  `json:"kubernetesVersion"`
 	UpgradeID         string                  `json:"upgradeID"`
+	Patches           PatchConfig             `json:"patches"`
```

Comment:
should this be a slice of patch configs? or is this literally supposed to be `[{//patchop}, {//patchop}, ...]`?

Reply:
ah never mind, the patch object is actually a list of operations.